portal/config/eproms/site_persistence_file.json

```diff
@@ -21,7 +21,6 @@
         "CUSTOM_PATIENT_DETAIL = True\n",
         "LOCALIZED_AFFILIATE_ORG = 'TrueNTH Global Registry'\n",
         "ORGS_W_CUSTOM_INVITES = ['TrueNTH Global Registry', 'IRONMAN']\n",
-        "MEDIDATA_RAVE_FIELDS = ['deceased', 'study_id', 'consent_status', 'dob', 'org']\n",
```

Comment:
Can you add `MEDIDATA_RAVE_ORG` and `MEDIDATA_RAVE_FIELDS` to  `config.py` so that they can be configured via environment variable ([eg `PIWIK_DOMAINS`](https://github.com/uwcirg/truenth-portal/blob/develop/portal/config/config.py#L117-L118))?